libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -365,3 +365,11 @@ _IRQL_requires_max_(PASSIVE_LEVEL) _Must_inspect_result_ ebpf_result_t
 
     return EBPF_SUCCESS;
 }
+
+BOOLEAN
+usersim_fault_injection_is_enabled(void)
+{
+    // Kernel mode replacement (for call from usersim).
+    // As of now, fault injection is not supported in kernel mode.
+    return FALSE;
+}
```

Comment:
FYI, this will interfere with my usersim changes, which compiles ebpf_platform_kernel.c (from a separate project) and links with usersim.  I'd recommend putting it into a separate .c file.

Reply:
Fixed.